src/test/java/duke/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,33 @@
+package duke.tasks;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeadlineTest {
+
+    // Testing the creation of a Deadline task and its default values.
+    @Test
+    public void createDeadlineTest() {
```

Comment:
Should this function just be called `createDeadline` as per the naming standard for tests as shown [here](https://se-education.org/guides/conventions/java/intermediate.html#naming:~:text=Underscores%20may%20be%20used%20in%20test%20method%20names%20using%20the%20following%20three%20part%20format%20featureUnderTest_testScenario_expectedBehavior())?